src/containers/Chat/ChatMessages/ChatMessages.tsx

```diff
@@ -582,37 +582,40 @@ export const ChatMessages: React.SFC<ChatMessagesProps> = ({ contactId, collecti
 
   if (conversationInfo && conversationInfo.messages && conversationInfo.messages.length > 0) {
     let reverseConversation = [...conversationInfo.messages];
-    reverseConversation = reverseConversation.map((message: any, index: number) => (
-      <ChatMessage
-        {...message}
-        contactId={contactId}
-        key={message.id}
-        popup={message.id === showDropdown}
-        onClick={() => showEditTagsDialog(message.id)}
-        setDialog={() => {
-          loadAllTags();
-
-          let messageTags = conversationInfo.messages.filter(
-            (messageObj: any) => messageObj.id === editTagsMessageId
-          );
-          if (messageTags.length > 0) {
-            messageTags = messageTags[0].tags;
+    reverseConversation = reverseConversation.map((message: any, index: number) => {
+      const key = index;
+      return (
+        <ChatMessage
+          {...message}
+          contactId={contactId}
+          key={key}
```

Comment:
Same here. Message id will always be unique. Why are we using index here?

Reply:
The issue was mock data, which basically had the same `message.id`